packages/legend-application-studio/src/stores/LegendStudioApplicationPlugin.ts

```diff
@@ -77,10 +77,10 @@ export type TestableMetadataGetter = (
   editorStore: EditorStore,
 ) => TestableMetadata | undefined;
 
-export type TestRunnerTabRenderer = (
-  selectedTab: string,
-  editorStore: EditorStore,
-) => React.ReactNode | undefined;
+export type TestRunnerTabSetup = (editorStore: EditorStore) => {
+  name: string;
```

Comment:
actually, let's split this to `id` and `title`, I think it's just nice to have id uniquely identified and title be something nice and well-formatted